pandas/tests/io/test_html.py

```diff
@@ -111,6 +111,58 @@ def spam_data(self, datapath):
     def banklist_data(self, datapath):
         return datapath("io", "data", "html", "banklist.html")
 
+    @pytest.fixture
+    def gh_13141_data(self):
```

Comment:
IMO I would inline these fixtures into the tests you created. I am not sure how sharable these fixtures will be as it seems specific to the github issue

Reply:
Only issue is that it's used in two separate tests. Any suggestions for a good workaround for that?

`gh_13141_expected` doesn't have to be a fixture though, so I'll put that into the test itself.

Reply:
Since the two tests are fairly small, I think it's okay if you combine the 2 tests to 1 single test.

Reply:
> Since the two tests are fairly small, I think it's okay if you combine the 2 tests to 1 single test.

The problem is that the first test is parameterized so merging them would mean that the second test will be run multiple times for no reason.

Reply:
Fixed by just removing the fixture dependency in the second test (the html used is irrelevant).